sources/api/pluto/src/eks.rs

```diff
@@ -46,8 +46,14 @@ where
     let config = sdk_config(region).await;
 
     let client = if let Some(https_proxy) = https_proxy {
-        let http_connector = proxy::setup_http_client(https_proxy, no_proxy)?;
```

Comment:
Would be good to flag this with a prominent `FIXME` since we can't merge without proxy support.

Reply:
Of course! I'll add it as a comment (I used `WIP` the commit message to signal that this isn't ready)